### PR TITLE
Fix 2 errors in the testing framework

### PR DESCRIPTION
**Pull request by obycode**
See #496.

### CODE REVIEW

File: deno/index.ts [hunk 1]
```diff
@@ -360,11 +360,11 @@ export namespace types {
   }
 
   export function ascii(val: string) {
-    return `"${val}"`;
+    return JSON.stringify(val);
   }
 
   export function utf8(val: string) {
-    return `u"${val}"`;
+    return `u${JSON.stringify(val)}`;
```
Comment (moodmosaic) [inline deno/index.ts:367]: Good catch here 👍

**Comment by moodmosaic** [comment]
Shall we re-run that fuzz test before merging?

**Review by moodmosaic — COMMENTED** [review]
Shall we re-run the fuzz test before merging?

**Comment by obycode** [comment]
> Shall we re-run the fuzz test before merging?

Yes, I've been running it and it looks good. You will need to restrict the string parameter a bit more to avoid `illegal ascii string` and `unknown escape character` errors, but I think this fixes the problems in the test infrastructure.

**Comment by obycode** [comment]
Example output with this fix:

```
Running file:///Users/brice/work/496-fuzz/tests/example_test.ts
Runtime error: example::set-str("") -> illegal ascii string ""
{ height: 2, receipts: [] }
{ height: 3, receipts: [ { result: '(ok " ")', events: [] } ] }
{ height: 4, receipts: [ { result: '(ok "!")', events: [] } ] }
Runtime error: example::set-str("\u0019") -> unknown escape character, 'u'
{ height: 5, receipts: [] }
{ height: 6, receipts: [ { result: '(ok " ")', events: [] } ] }
{ height: 7, receipts: [ { result: '(ok "%")', events: [] } ] }
{ height: 8, receipts: [ { result: '(ok """)', events: [] } ] }
{ height: 9, receipts: [ { result: '(ok """)', events: [] } ] }
{ height: 10, receipts: [ { result: '(ok "2")', events: [] } ] }
{ height: 11, receipts: [ { result: '(ok """)', events: [] } ] }
* [Exampe] ascii fuzz-test ... ok (32ms)
```

**Comment by moodmosaic** [comment]
Sounds good. Yes, making the infrastructure more robust is the ultimate goal. Thank you 👍

On that fuzz test, we can tweak the ascii generator if needed and see what happens. 🎲

**Comment by moodmosaic** [comment]
Might be out of the scope of this PR, but while we're here try to re-run your test this time by changing numRuns: 10 to numRuns: 5000. See if anything else breaks.

**Comment by obycode** [comment]
I just tried this (without any modifications to the ascii generator). The test finishes successfully, with a bunch of runtime errors seen from illegal characters.

**Comment by moodmosaic** [comment]
👍 Thanks @obycode

**Review by lgalabru — APPROVED** [review]
Thanks @obycode!

**Comment by obycode** [comment]
Not sure why those two "Generate test coverage" jobs didn't run. I'm going to try re-running. @CharlieC3, do you have any thoughts? Has this new check run at all before?

**Comment by obycode** [comment]
Those jobs don't show here - https://github.com/hirosystems/clarinet/actions/runs/2721283881/usage

**Comment by obycode** [comment]
Oh, I see, those status bits were set, but this branch doesn't have those jobs. Rebasing on top of main now...

**Comment by CharlieC3** [comment]
@obycode Looks like those jobs are commented out in the workflow in both branches?
https://github.com/hirosystems/clarinet/blob/fix/496-sender/.github/workflows/ci.yaml#L91-L165
https://github.com/hirosystems/clarinet/blob/develop/.github/workflows/ci.yaml#L91-L165

**Comment by obycode** [comment]
Oh, you're right... what was I just looking at?

Ahh, in the in-progress `feat/ls-vscode-commands` they're un-commented. I'll just disable those checks from the required list for now.

**Pull request merged**